### PR TITLE
rawagner_ViewMenu cannot manipulate with check styled menus

### PR DESCRIPTION
**Pull request by tsedmik**
Example: I'm unable to change (deselect) _Activate on new events_ menu item of _Error Log_ view.

### CODE REVIEW

**Comment by vpakan** [comment]
Can one of the admins verify this patch? (use 'ok to test' comment to run verification job)

**Comment by rawagner** [comment]
Fix the failing test please

**Comment by tsedmik** [comment]
Fixed. Can one of the admins verify this patch?

File: plugins/org.jboss.reddeer.swt/src/org/jboss/reddeer/swt/impl/menu/ViewMenu.java [hunk 1]
```diff
@@ -64,7 +68,18 @@ public void select() {
 		if (item == null) {
 			mh.select(menuItem);
 		} else {
-			mh.select(item);
+			if ((item.getAction().getStyle() == IAction.AS_CHECK_BOX) || (item.getAction().getStyle() == IAction.AS_RADIO_BUTTON)) {
```
Comment (rawagner) [inline plugins/org.jboss.reddeer.swt/src/org/jboss/reddeer/swt/impl/menu/ViewMenu.java]: Looks good but we prefer all sync and async operations to be executed within Handler classes. Could you please move this whole logic to MenuHandler class https://github.com/tsedmik/reddeer/blob/REDDEER-1359/plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/handler/MenuHandler.java#L70 ? Thanks!

**Comment by tsedmik** [comment]
Sync operation was moved into _MenuHandler_ class.

**Comment by rawagner** [comment]
pushed to master, thanks!

**Pull request closed without merging**